setup.py

```diff
@@ -180,7 +180,7 @@ def run(self):
         'PyInstaller': ['bootloader/*/*'],
         # This file is necessary for rthooks (runtime hooks).
         'PyInstaller.loader': ['rthooks.dat'],
-        },
+    },
```

Comment:
No white-space change, please.

Reply:
It looks like you fixed this during the rebase.